packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getLeaves.st

```diff
@@ -0,0 +1,13 @@
+information extraction
+getLeaves 
+	
+	| leaves |
+	
+	leaves := OrderedCollection new.
+
+	self getChildrenWithRelevantPerformanceImpact do: [ :child |
```

Comment:
I would suggest renaming the variable `child` to `node`. It is the node currently being looked at and the child relation to the node's parent is not of importance in the block closure.

Reply:
Aus meiner Sicht ist der Lesefluss durch die Verwendung von `child`etwas besser. Beim lesen des Codes ist besser ersichtlich das ich vom jeweiligen Knoten die Kinder betrachte und auf diesen die Methode `getLeaves` rekursiv aufrufe.